AsyncDisplayKit/ASCollectionView.mm

```diff
@@ -405,6 +421,11 @@ - (void)moveItemAtIndexPath:(NSIndexPath *)indexPath toIndexPath:(NSIndexPath *)
   [_dataController moveRowAtIndexPath:indexPath toIndexPath:newIndexPath withAnimationOptions:kASCollectionViewAnimationNone];
 }
 
+- (NSString *)__reuseIdentifierForKind:(NSString *)kind
+{
+  return [NSString stringWithFormat:@"_ASCollectionSupplementaryView_%@", kind];
```

Comment:
Also small optimization, but it is close to 10x faster to do something like [@"_ASCollectionSupplementaryView_%@" stringByAppendingString:kind].